### ISSUE TITLE
C: nl_after_struct affects prototype definitions including struct parameters

### ISSUE THREAD

**Issue by jaguarondi** [uncrustify/uncrustify#2616]
The setting `nl_after_struct = 2` will affect prototype definitions with struct in the parameter list.

This:
```
void ethernetif_input(struct netif netif);
void ethernetif_update_config(struct netif netif);
```
becomes this:
```
void ethernetif_input(struct netif netif);

void ethernetif_update_config(struct netif netif);
```

- Uncrustify-0.70.1_f
- https://pastebin.com/hTSj0Ubx

**Comment by guy-maurel** [comment]
PR #2619 is a bugfix proposal.
Please test it and report.

**Comment by jaguarondi** [comment]
Tested and the issue is fixed  in PR #2619.
